File: week_10/c_sharp_course/week_1/day_4/composition/starter_code/wizard_management/MagicCarpet.cs
namespace WizardManagement {
  public  class MagicCarpet : Carpet 
  {
    public MagicCarpet(string colour) : base(colour)
    {

    }

    public string Fly()
    {
      return "Hovering up, straightening out, flying off!";
    }
  }
}

File: week_06/day_3/composition/Composition_Lesson_End/app/src/main/java/example/codeclan/com/composition_lesson_end/wizard_management/MagicCarpet.java
package example.codeclan.com.composition_lesson_end.wizard_management;

import example.codeclan.com.composition_lesson_end.behaviours.Flyable;

/**
 * Created by user on 28/08/2017.
 */

public class MagicCarpet extends Carpet implements Flyable {

    public MagicCarpet(String colour){
        super(colour);
    }

    public String fly(){
        return "Hovering up, straightening out, flying off!";
    }

}
